packages/pn-pa-webapp/src/pages/AppStatus.page.tsx

```diff
@@ -0,0 +1,145 @@
+  }, [dispatch, getCurrentAppStatus]);
+
+  const fetchDowntimeLog = useCallback(() => {
+    const fetchParams: GetDowntimeHistoryParams = { 
+      startDate: "1900-01-01T00:00:00Z",
+      functionality: [KnownFunctionality.NotificationCreate, KnownFunctionality.NotificationVisualization, KnownFunctionality.NotificationWorkflow],
+      size: String(paginationData.size),
+      page: paginationData.resultPages[paginationData.page],
+    };
+    void dispatch(getDowntimeLogPage(fetchParams));
+  }, [dispatch, getDowntimeLogPage, paginationData.page, paginationData.size]);
+
+  const fetchDowntimeLegalFactDocumentDetails = useCallback(
+    (legalFactId: string) => void dispatch(getDowntimeLegalFactDocumentDetails(legalFactId)), 
+  [dispatch, getDowntimeLegalFactDocumentDetails]);
+
+  useEffect(() => {
+    if (!isInitialized) {
+      dispatch(clearPagination());
+      setIsInitialized(true);
+    }
+  }, [isInitialized]);
+
+  /*
+   * whenever fetchDowntimeLog changes (e.g. when pagination parameters change)
+   * I decide to perform the status API call along with that fo the downtime log
+   * to bring the user status information as updated as possible.
+   * -------------------------------
+   * Carlos Lombardi, 2022.11.11
+   */
+  useEffect(() => {
+    if (isInitialized) {
+      fetchCurrentStatus();
+      fetchDowntimeLog();
+    }
+  }, [fetchCurrentStatus, fetchDowntimeLog, isInitialized]);
+
+  const handleChangePage = (paginationData: PaginationData) => {
+    dispatch(setPagination({ size: paginationData.size, page: paginationData.page }));
+  };
+
+  const lastCheckTimestampFormatted = useMemo(() => {
+    if (currentStatus) {
+      const dateAndTime = formatDateTime(currentStatus.lastCheckTimestamp);
+      return `${dateAndTime.date}, ${dateAndTime.time}`;
+    } else {
+      return "";
+    }
+  }, [currentStatus]);
+
+  // resultPages includes one element per page, *including the first page*
+  // check the comments in the reducer
+  const totalElements = paginationData.size * paginationData.resultPages.length;
+  // paginagion will show just a single page number, along with the arrows
+  // because the lookup size of the API is just one, there is no (easy) way to know 
+  // whether there is more than one page forward
+  const pagesToShow = [paginationData.page + 1];    
+
+  return <Box p={3}>
+    <Stack direction="column">
+
+      {/* Titolo status */}
+      <TitleBox
+        title={t('appStatus.title')} variantTitle='h4'
+        subTitle={t('appStatus.subtitle')} variantSubTitle='body1'
+      />
+
+      {/* Dati relativi al status */}
+      <ApiErrorWrapper apiId={APP_STATUS_ACTIONS.GET_CURRENT_STATUS} reloadAction={() => fetchCurrentStatus()} mt={3}>
+        {currentStatus && <AppStatusBar status={currentStatus} />}
+        { currentStatus &&
+          <Stack direction="row" justifyContent="center">
+            <Typography variant="caption" sx={{ mt: 2, color: theme.palette.text.secondary }}>
```

Comment:
Also here maybe we can avoid the use of useTheme

Reply:
done, both in pa-webapp and pf-webapp